docs/data/routes/docs/nextjs/getting-started/options/en.md

```diff
@@ -3,4 +3,24 @@ name: options
 routeTemplate: ./data/component-templates/article.yml
 title: Getting Started options overview
 ---
-# Getting Started options overview
\ No newline at end of file
+#  Development environment options for JSS Next.js projects
+
+Sitecore offers two paths for setting up your development environment based on your role, workflow, and operating system. 
+
+## Full-stack development
+
+Sitecore provides a Getting Started template for Sitecore developers that want to try out  Sitecore JSS with [Docker](https://www.docker.com/), [Sitecore Containers](https://containers.doc.sitecore.com/), the [Sitecore Next.js SDK](https://jss.sitecore.com/), and [Sitecore Content Serialization](https://doc.sitecore.com/developers/100/developer-tools/en/sitecore-content-serialization.html).
```

Comment:
IIRC, we use relative URLs for linking across docs so that it works correctly on local and staging environment

Reply:
Are you referring to https://jss.sitecore.com/? 

That would change to `/`. How about the other paths? Will we use `/docs/nextjs/{etc}`?